### DISCUSSION TITLE
Discussion topics for Apr 11, 2022 Engineering Sync

### DISCUSSION THREAD

**Discussion by carlosfelix2** [CDCgov/prime-reportstream#5099] [Show and tell]
Discussion topics for Apr 11, 2022 Engineering Sync

**Comment by bgantick** [comment]
Manual testing plans for staging: in [SharePoint](https://cdc.sharepoint.com/:w:/r/teams/USDSatCDC/_layouts/15/Doc.aspx?sourcedoc=%7B0C859459-8C1C-413D-BD3D-B1162BB6D45F%7D&file=ReportStream%20Staging%20Manual%20Smoke%20Test%20Plans.docx&action=default&mobileredirect=true&cid=bd1cdb86-4863-4f77-9aee-1ea67a6f6e71) as a reference.

**Reply by RickHawesUSDS** [reply]
STLT integration tests and Preview
